pkgs/applications/video/pitivi/default.nix

```diff
@@ -74,6 +75,13 @@ python3.pkgs.buildPythonApplication rec {
     librosa
   ];
 
+  preFixup = ''
+    gappsWrapperArgs+=(
+      # The icon theme is hardcoded.
+      --prefix XDG_DATA_DIRS : "${hicolor-icon-theme}/share"
+      )
+    '';
```

Comment:
```suggestion
    )
  '';
```

Reply:
Are you using anything to format the nix files ? In order so I can fix this for once and for all ?

Reply:
I use nixpkgs-fmt, but the choice shouldn't matter too much. The aim is to ensure consistency of the format in the file, with `postPatch` for example.